Dockerfile

```diff
@@ -53,6 +60,9 @@ RUN mkdir -p /evergreen
 COPY client /evergreen/client
 COPY essentials.yaml /evergreen
 
+# FIXME REMOVE (to ease iteration/speed just for now), see also shim-startup-wrapper.sh
+RUN wget --quiet http://mirrors.jenkins.io/war-stable/latest/jenkins.war -O /usr/share/jenkins/jenkins.war
```

Comment:
complementary to the shim. The goal here is to try and leverage Docker cache and avoid having to download the WAR **each** time one would start the container. Which is already a few seconds on a high bandwidth network, but can kill everything on a more flaky network when hacking on evergreen.